src/AssetBundle.php

```diff
@@ -16,10 +16,10 @@
  * An asset bundle can depend on other asset bundles. When registering an asset bundle with a view, all its dependent
  * asset bundles will be automatically registered.
  *
- * @psalm-import-type CssFile from AssetManager
- * @psalm-import-type CssString from AssetManager
- * @psalm-import-type JsFile from AssetManager
- * @psalm-import-type JsString from AssetManager
+ * @phpstan-import-type CssFile from AssetManager
```

Comment:
Do we have to change it?

Reply:
Do you think we should keep Psalm annotations? PHPStan can read it.

Reply:
It's fine either way. Just too many changes for no particular practical gain.

Reply:
If we keep psalm annotations, it can be useful to users who use psalm.